### DISCUSSION TITLE
Welcome to vite-plugin-federation Discussions!

### DISCUSSION THREAD

**Discussion by flyfishzy** [originjs/vite-plugin-federation#113] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by roshan2921** [comment]
@flyfishzy @ruleeeer Hey there, Im trying to share axios interceptors between a  remote and host (Interceptors in host not in remote). Here is the sample vite.config.ts code of remote

export default defineConfig((options) => {
    return {
        base: 'my-root-folder',
        build: {
            outDir: 'my-root-folder',
            emptyOutDir: true,
            manifest: true,
            polyfillModulePreload: false,
            assetsInlineLimit: 40960,
            target: "esnext",
            minify: false,
            cssCodeSplit: false,
            rollupOptions: {
                input: {
                    MYCOMPONENTSNAME: 'MYCOMPONENTS HERE'
                }
            }
        },
        server: {
            hmr: {
                protocol: 'ws',
                host: 'localhost'
            }
        },
        plugins: [
            createVuePlugin(),
            vueSvgPlugin(),
            federation({
                name: "remote-app",
                filename: "remoteEntry.js",
                exposes: {
                    //exposing some components 
                },
                remotes: {},
                shared: ["vue", "axios"],           //Here Im trying to share my axios from remote to shell
            }),
        ]
    })

You can see the error message here:
![Error message](https://billapp.blob.core.windows.net/bills-processed/cjsError.png)
